### ISSUE TITLE
Contents of all panels erased in w2layout on IE11

### ISSUE THREAD

**Issue by Altair68** [vitmalina/w2ui#1117]
On line 7185 there's this code
```javascript
$(obj.box)
    .attr('name', obj.name)
    .addClass('w2ui-layout')
    .html('<div></div>');
```
On IE11 the call of `.html()` causes all of the content of the panels to be erased.

**Comment by vitmalina** [comment]
Isn't it what suppose to happen? Same in Chrome/FF?

**Comment by Altair68** [comment]
Yep, actually that is what the code suggests. In FF the panels keep their content, so it can be set by the `refresh()` method, the IE deletes even that content so the `refresh()` method cannot set it back. Unfortunately I can't test it on Chrome.

**Comment by vitmalina** [comment]
The code seems transparent. I am not sure I understand what is going on. Why obj.panels are created by IE?

**Comment by Altair68** [comment]
If I debug into the Code the variable `obj` holds all of the information that was on the page before the layout-method was called, including the contents of all panels I defined. When I step over this function in FF all panels are still filled with their content. If I do the same on IE the method mentioned before erases all attributes and contents from the panels inside the variable `obj`

**Comment by vitmalina** [comment]
Can you please create a jsFiddle with the problem. I will run IE in a VM and look at it.

**Comment by Altair68** [comment]
Guess this should do:
    http://jsfiddle.net/gd8zjh3o/1/

I'm not too sure the problem occurs there, can't test it online with IE due to company restrictions...
